packages/fury-adapter-oas3-parser/lib/parser/oas/parseOauthFlowObject.js

```diff
@@ -0,0 +1,62 @@
+const R = require('ramda');
+const {
+  isExtension, hasKey, getValue,
+} = require('../../predicates');
+const {
+  createInvalidMemberWarning,
+} = require('../annotations');
+const pipeParseResult = require('../../pipeParseResult');
+const parseObject = require('../parseObject');
+const parseString = require('../parseString');
+
+const name = 'Oauth Flow Object';
+const requiredKeys = ['scopes'];
+
+/**
+ * Parse Oauth Flow Object
+ *
+ * @param namespace {Namespace}
+ * @param element {Element}
+ * @returns ParseResult
+ *
+ * @see https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#oauthFlowObject
+ * @private
+ */
+function parseOauthFlowObject(context, object) {
+  const { namespace } = context;
+  const scopesName = `${name}' 'scopes`;
+
+  const parseScopeMember = R.cond([
+    [R.T, parseString(context, scopesName, false)],
+  ]);
+
+  const parseScopes = pipeParseResult(namespace,
+    parseObject(context, scopesName, parseScopeMember, [], true),
```

Comment:
This will need rebasing on master it would currently fail and should be changed once rebased:

```suggestion
    parseObject(context, scopesName, parseScopeMember, [], [], true),
```

Reply:
```diff
diff --git a/packages/fury-adapter-oas3-parser/lib/parser/oas/parseOauthFlowObject.js b/packages/fury-adapter-oas3-parser/lib/parser/oas/parseOauthFlowObject.js
index 3566671..98f6ff0 100644
--- a/packages/fury-adapter-oas3-parser/lib/parser/oas/parseOauthFlowObject.js
+++ b/packages/fury-adapter-oas3-parser/lib/parser/oas/parseOauthFlowObject.js
@@ -31,7 +31,7 @@ function parseOauthFlowObject(context, object) {
   ]);
 
   const parseScopes = pipeParseResult(namespace,
-    parseObject(context, scopesName, parseScopeMember, [], true),
+    parseObject(context, scopesName, parseScopeMember, [], [], true),
     scopes => new namespace.elements.Array(scopes.content),
     R.map((member) => {
       const scope = member.key.clone();
@@ -54,7 +54,7 @@ function parseOauthFlowObject(context, object) {
   ]);
 
   const parseOauthFlow = pipeParseResult(namespace,
-    parseObject(context, name, parseMember, requiredKeys, true));
+    parseObject(context, name, parseMember, requiredKeys, [], true));
 
   return parseOauthFlow(object);
 }
```